source/functions/initializeWecSim.m

```diff
@@ -532,20 +503,12 @@
     waves(iW).listInfo();
 end; clear iW
 fprintf('\nList of Body:\n ');
-fprintf('Number of Hydro Bodies = %u \n',simu.numHydroBodies)
-for i = 1:simu.numHydroBodies
-    if body(i).nonHydro == 0
-        body(i).listInfo(i,'0')
-    end
-end;  clear i
-if numNonHydroBodies ~= 0
-    fprintf('\nNumber of Non-Hydro Bodies = %u \n',numNonHydroBodies)
-    for i = 1:(numNonHydroBodies+simu.numHydroBodies)
-        if body(i).nonHydro == 1
-            body(i).listInfo(i,'1')
-        end
-    end
-end; clear i
+fprintf('\nNumber of Hydro Bodies = %u \n',simu.numHydroBodies)
+fprintf('\nNumber of Drag Bodies = %u \n',numDragBodies)
+for i = 1:length(body)
```

Comment:
Number of drag bodies should also be removed

Reply:
The number of drag bodies is still relevant. We previously enumerated hydro, nonhydro, and drag bodies. Now we just enumerate hydro and drag